blog/_posts/2018-07-09-$we-init-introductory-post.md

```diff
@@ -0,0 +1,74 @@
+# Dreams come true !
```

Comment:
Hi @Rahmeen14 ! Thanks for the PR. Please check the other blog posts for the proper markdown format. We need the whole set of parameters at the start.

Reply:
Hi @Rahmeen14 ! Permalink should be the same as the file name. Please check other files for the example. Everything else looks good to me.